### PR TITLE
Trend Feature PR implementation

### PR DESCRIPTION
**Pull request by arturgvieira**
Hi Mobius Team, I have just finished the rough draft of the Trend feature I was speaking about in the Issue I opened earlier.

Here are a couple snapshots of the end result.

screen shot 2018-10-14 at 9 42 10 am
screen shot 2018-10-14 at 9 44 39 am

Ok, I look forward to your review. Thank you.

### CODE REVIEW

**Comment by arturgvieira** [comment]
It has been over 6 months that I don't play around with Redux, so I am not sure if I am missing anything. Ok thanks.

File: src/components/shared/Icon/Icon.js [hunk 1]
```diff
@@ -12,6 +12,9 @@ const icons = {
   paymentSent: require('./images/payment-sent.png'),
   paymentReceived: require('./images/payment-received.png'),
   paymentAccount: require('./images/payment-received.png'),
+
+  decorativeArrowUp: require('./images/arrow-up.png'),
```
[the remark this answers is not in the document]
Comment (lysyi3m) [reply src/components/shared/Icon/Icon.js]: You can easily customize their color via styling. Take a look here for example https://github.com/mobius-network/wallet/blob/f9c4dfd6db0737821971b85f36689711b8202477/src/components/App/Send/AddressForm/Header/styles.js#L32-L37

File: src/state/prices/selectors.js [hunk 2]
```diff
@@ -54,3 +54,12 @@ export const getUsdBalance = createSelector(
     return total.toFixed(2);
   }
 );
+
+export const getAssetChange = createSelector(getPrices, prices => Object.keys(prices).reduce((acc, asset) => {
+  acc[asset] = {
+    amount: prices[asset].change,
+    icon:
+        prices[asset].change > 0 ? 'decorativeArrowUp' : 'decorativeArrowDown',
```
[the remark this answers is not in the document]
Comment (lysyi3m) [reply src/state/prices/selectors.js]: > you prefer the logic on the component side?

In short - yes. We're using Redux to store data and selectors to read this data from the store. So it's an unexpected behavior, when selector returns extra UI-specific things like icon names (or class names in normal web app). 

You can detect proper icon name during render of component itself. Something like this (quick example):

```jsx

class AmountItem extends Component { 
  ...
 
  renderChangeIcon() {
    const { change } = this.props;
   
    const changeIconName = change > 0 ? 'one' : 'another';
    
    return <IconChangetype name={changeIconName} />
  } 

  ...

  render (
    ...
    {this.renderChangeIcon()}
  )
}
```

**Comment by arturgvieira** [comment]
@lysyi3m Ok, thank you. I have made the corrections requested.

**Comment by arturgvieira** [comment]
Please let me know if there is anything else. Ok, look forward to the review.

**Comment by arturgvieira** [comment]
Great stuff, thank you for the opportunity! Hey, I am available to make some more, please open issues and tag them as "Help Wanted", I'll be on the lookout for them.

**Pull request merged**